fvm/handler/programs_test.go

```diff
@@ -23,17 +23,17 @@ func Test_Programs(t *testing.T) {
 	addressC := flow.HexToAddress("0c")
 
 	contractALocation := common.AddressLocation{
-		Address: common.Address(addressA),
```

Comment:
I'm a bit confused why all these got changed back, is it possibly a single commit is the culprit?